pkg/gatewayserver/io/basicstationlns/basicstationlns_test.go

```diff
@@ -118,7 +124,7 @@ func TestClientTokenAuth(t *testing.T) {
 			a := assertions.New(t)
 			h := http.Header{}
 			h.Set("Authorization", fmt.Sprintf("%s%s", tc.TokenPrefix, tc.AuthToken))
-			conn, _, err := websocket.DefaultDialer.Dial("ws://localhost:8100/api/v3/gs/io/basicstation/traffic/"+tc.GatewayID, h)
+			conn, _, err := websocket.DefaultDialer.Dial(servAddr+connectionRootEndPoint+tc.GatewayID, h)
```

Comment:
I would just put string literals here and/or build the string explicitly with `fmt.Sprintf` but ok